doc/source/whatsnew/v3.0.0.rst

```diff
@@ -24,6 +24,10 @@ enhancement1
 enhancement2
 ^^^^^^^^^^^^
 
+New Deprecation Policy
+^^^^^^^^^^^^^^^^^^^^^^
+pandas 3.0.0 introduces a new 3-stage deprecation policy: using ``DeprecationWarning`` initially, then switching to ``FutureWarning`` for broader visibility in the last minor version before the next major release, and then removal of the deprecated functionality in the major release.
```

Comment:
It might be good to add some reasoning to this
(e.g. This was done to give downstream packages more time to adjust to pandas deprecations, which should reducing the amount of warnings that an user gets from code that isn't theirs.)

Reply:
I think we'd have to also say that there would always be a minimal amount of time  (6 months, maybe more??) between the last minor version before the next major release.

Reply:
> a minimal amount of time (6 months, maybe more??) between the last minor version before the next major release.

Is this consistent? 3.0 was planned to be released within 6 months of releasing 2.2.0 (#57064)

Reply:
> > a minimal amount of time (6 months, maybe more??) between the last minor version before the next major release.
> 
> Is this consistent? 3.0 was planned to be released within 6 months of releasing 2.2.0 (#57064)

In your last commit, you wrote "This was done to give downstream packages more time to adjust to pandas deprecations".  So if we are going to have a policy about this, then we have to decide how much time is sufficient for "more time".